source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,82 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+In this guide, we show you how to skip returned results.
```

Comment:
I think it could be helpful to be more specific here.

```suggestion
In this guide, we show you how to skip a specified number of returned results.
```